docker/Dockerfile

```diff
@@ -32,12 +33,21 @@ RUN apk update && \
       yaml-dev \
       zlib-dev \
       ncurses-dev \
-      bison \
-      autoconf \
-    && echo "gem: --no-ri --no-rdoc" > /etc/gemrc \
-    && bundle install --system $BUNDLER_ARGS \
-    && apk del .ruby-builddeps \
-    && rm -rf /var/cache/apk/*
+    && echo "gem: --no-ri --no-rdoc" > /etc/gemrc
```

Comment:
The problem here is that not all gems are accessible for non-root user after `bundle install --system`:
```
/usr/local/bundle/gems/activesupport-4.2.8/lib/active_support/dependencies.rb:274:in `require': cannot load such file -- robots (LoadError)
```
Probably I can fix this by setting the right permissions to whole `/usr/local/bundle/gems/`